tests/setup/testUtils.js

```
@@ -0,0 +1,106 @@
+ * Test utilities for QQ Music API tests
+ */
+
+const path = require('path');
+const fs = require('fs');
+
+/**
+ * Load test fixture from file
+ * @param {string} fixtureName - Name of the fixture file
+ * @returns {any} Parsed fixture data
+ */
+function loadFixture(fixtureName) {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)